### PR TITLE
Add doc to migrate from OC 10.5 to NC 20.0

### PR DESCRIPTION
**Pull request by PVince81**

### CODE REVIEW

File: admin_manual/maintenance/migrating_owncloud.rst [hunk 1]
```diff
@@ -34,8 +36,21 @@ See the table below for a version map, where migrating is easily possible:
...
-After downloading the correct version of Nextcloud from our
-`older releases page <https://nextcloud.com/changelog/>`_,
-proceed like described in the :doc:`manual_upgrade` manual.
+1. First download the correct version of Nextcloud from our `older releases page <https://nextcloud.com/changelog/>`_,
+
+2. Make sure to have do a :doc:`backup<backup>` before migrating.
+
+3. Follow the upgrade instructions described in the :doc:`manual_upgrade` manual.
+
+4. When migrating to Nextcloud 20.0 or later, you will also need to run the following commands after ``occ upgrade``:
```
Comment (ChristophWurst) [inline admin_manual/maintenance/migrating_owncloud.rst:45]: optionally link to the occ page for those who don't know how to run occ

**Review by ChristophWurst — APPROVED** [review]
Looks good!

File: admin_manual/maintenance/migrating_owncloud.rst (hunk 1 above)
Comment (PVince81) [reply admin_manual/maintenance/migrating_owncloud.rst:45]: If they have followed the previous step and went to "manual_upgrade" they should already know.

**Comment by MorrisJobke** [comment]
@PVince81 @ChristophWurst Backport to 19 and 20?

**Comment by ChristophWurst** [comment]
sure, why not

**Pull request merged**